pom.xml

```diff
@@ -43,6 +50,49 @@
         </pluginRepository>
     </pluginRepositories>
 
+    <properties>
+        <workflow.version>1.8</workflow.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>${workflow.version}</version>
+            <scope>test</scope>
```

Comment:
Also possible, though this carries some complications, for example that you need a `test`-scoped dependency on `git-server` too. I preferred to limit the dependencies to those which are actually being used.